docs/storage.md

```diff
@@ -5,19 +5,26 @@ sort_rank: 5
 
 # Storage
 
-Prometheus includes a local on-disk time series database, but also optionally integrates with remote storage systems.
+Prometheus includes a local time series database, but also optionally integrates with remote storage systems.
 
 ## Local storage
 
-Prometheus's local time series database stores time series data in a custom format on disk.
+Prometheus's local time series database stores data in a custom, highly efficient format on local storage.
 
 ### On-disk layout
 
 Ingested samples are grouped into blocks of two hours. Each two-hour block consists of a directory containing one or more chunk files that contain all time series samples for that window of time, as well as a metadata file and index file (which indexes metric names and labels to time series in the chunk files). When series are deleted via the API, deletion records are stored in separate tombstone files (instead of deleting the data immediately from the chunk files).
 
-The block for currently incoming samples is kept in memory and not fully persisted yet. It is secured against crashes by a write-ahead log (WAL) that can be replayed when the Prometheus server restarts after a crash. Write-ahead log files are stored in the `wal` directory in 128MB segments. These files contain raw data that has not been compacted yet, so they are significantly larger than regular block files. Prometheus will keep a minimum of 3 write-ahead log files, however high-traffic servers may see more than three WAL files since it needs to keep at least two hours worth of raw data.
+The current block for incoming samples is kept in memory and is not fully
+persisted. It is secured against crashes by a write-ahead log (WAL) that can be
+replayed when the Prometheus server restarts. Write-ahead log files are stored
+in the `wal` directory in 128MB segments. These files contain raw data that
+has not yet been compacted; thus they are significantly larger than regular block
+files. Prometheus will retain a minimum of 3 write-ahead log files.
+High-traffic servers may retain more than three WAL files in order to to keep at
```

Comment:
We're inconsistent here between 3 and three

Reply:
Agreed, adding that change to the pre-existing text.